### PR TITLE
New package: SpyShelter.SpyShelter version 15.0.0.480

### LINKED ISSUE #138588: [Package Request]: SpyShelter
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://spyshelter.com
Publisher: Netmeetings LLC
Package Name: SpyShelter
Description: Reveal hidden behavior of sneaky Windows processes on your PC, then take control of them with SpyShelter's powerful application control rules.  Detect and stop hidden malware.
Package Version: 15.0.0.480
Installer URL: https://spyshelter.com/files/SpyShelterSetup.exe

Thank you

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---
- Resolve: https://github.com/microsoft/winget-pkgs/issues/138588

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138664)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 720.)

**Review by Trenly — APPROVED** [review]

File: manifests/s/SpyShelter/SpyShelter/15.0.0.480/SpyShelter.SpyShelter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: SpyShelter.SpyShelter
+PackageVersion: 15.0.0.480
+InstallerType: nullsoft
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+Installers:
+- InstallerUrl: https://spyshelter.com/files/SpyShelterSetup.exe
+  Architecture: x86
```
Comment (Trenly) [inline manifests/s/SpyShelter/SpyShelter/15.0.0.480/SpyShelter.SpyShelter.installer.yaml:12]:
```suggestion
  Architecture: x64
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Clicked the wrong button - Seems to install fine on my x64 system

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 7:43:34 PM: 
> 2024-02-12 19:43:34.049 [CLI ] Package hash verification failed. SHA256 in manifest [f9c2d3f96e9a95bd277f512c236ec6e8b8994a35693ab758e033f7ac6bf39b2f] does not match download [458bc378f4e522b4b28f8ce24cc632dfa5795b3af091d4b6c4c4a78ba4f81917]

(Automated response - build 729.)

File: manifests/s/SpyShelter/SpyShelter/15.0.0.480/SpyShelter.SpyShelter.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: SpyShelter.SpyShelter
+PackageVersion: 15.0.0.480
+InstallerType: nullsoft
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+Installers:
+- InstallerUrl: https://spyshelter.com/files/SpyShelterSetup.exe
+  Architecture: x64
+  InstallerSha256: F9C2D3F96E9A95BD277F512C236EC6E8B8994A35693AB758E033F7AC6BF39B2F
```
Comment (stephengillie) [inline manifests/s/SpyShelter/SpyShelter/15.0.0.480/SpyShelter.SpyShelter.installer.yaml:13]:
```suggestion
  InstallerSha256: 458BC378F4E522B4B28F8CE24CC632DFA5795B3AF091D4B6C4C4A78BA4F81917
```

**Comment by stephengillie** [comment]
https://github.com/microsoft/winget-pkgs/pull/138664#discussion_r1487131099

**Review by stephengillie — APPROVED** [review]

**Pull request merged**